hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/codec/ContainerReplicaHistoryListCodec.java

```diff
@@ -39,7 +39,7 @@
     implements Codec<ContainerReplicaHistoryList> {
 
   // UUID takes 2 long to store. Each timestamp takes 1 long to store.
-  static final int SIZE_PER_ENTRY = 4 * Long.BYTES;
+  static final int SIZE_PER_ENTRY = 5 * Long.BYTES;
```

Comment:
Where is this used, is it in a map? Do we need to account for reference overheads?

Reply:
We don't need to, since we are storing as pure Longs. This is used in the Codec.

Reply:
I am planning on changing the serialize/desereialize to a proto based logic to allow us to add more fields in the future without worrying about compatibility.

Reply:
I am planning to do it in a follow up patch in this same PR.